src/components/BookmarkList/BookmarkList.tsx

```diff
@@ -97,39 +97,43 @@ type BookmarkListProps = {
   userCollectionOptions?: Collection[]
   handleAddToCollection: (b: BookmarkRecord, c?: ObjectId) => void
   canAddNewCollection?: boolean
+  className?: string
 }
 
 const BookmarkList = ({
   bookmarks,
   userCollectionOptions = [],
   handleAddToCollection,
   canAddNewCollection = true,
+  className = '',
 }: BookmarkListProps) => {
   const filterInvalidBookmarks = (
     b: Partial<BookmarkRecord>
   ): b is BookmarkRecord => !(b.id === undefined || b.url === undefined)
 
   return (
-    <Table striped fullWidth>
-      <thead>
-        <tr>
-          <th scope="col">Application name</th>
-          <th scope="col">Description</th>
-          <th></th>
-        </tr>
-      </thead>
-      <tbody>
-        {bookmarks.filter(filterInvalidBookmarks).map((b) => (
-          <BookmarkListRow
-            key={`bookmark_${b.id}`}
-            bookmark={b}
-            userCollectionOptions={userCollectionOptions}
-            handleAddToCollection={handleAddToCollection}
-            canAddNewCollection={canAddNewCollection}
-          />
-        ))}
-      </tbody>
-    </Table>
+    <div className={className}>
+      <Table striped fullWidth>
```

Comment:
question: I'm curious why did we need a new div here vs applying className to `Table`?

Reply:
`<Table>` is a react uswds component and wouldn't accept a `className` prop. If I missed something and there's a way to pass one, please let me know!

Reply:
Sadly I don't know all the React-uswds components well enough yet. But I checked and looks like it does not accept any classes. I'm not sure if it should or not, but does answer my question :)